server/api/profiles.js

```diff
@@ -79,7 +79,7 @@ router.get("/applicant/:applicant_id", async (req, res) => {
     let profile = await getProfileByApplicantId(id);
 
     if (profile == null) {
```

Comment:
this it may come as an empty array too. you need to check that.
e.g ` if (profile === null || !profile.length)`

Reply:
Changed. Thanks for your assistance.